hw/ip/entropy_src/doc/_index.md

```diff
@@ -10,82 +10,95 @@ This module conforms to the [Comportable guideline for peripheral functionality.
-A read of the {{< regref "ES_THRESH" >}} register could be done to verify how many entries are in the FIFO at the current time.
-The {{< regref "ES_THRESH" >}} register will determine when the status bit is set, or when the interrupt will assert if enabled.
-For this revision, this same {{< regref "ES_THRESH" >}} register has a range of between 1 and 4 entries in the TLUL FIFO.
-
-For hardware consumption of entropy, if the HWIF FIFO is not empty, and the entropy consuming hardware is ready, then entropy bits will be transferred over the hardware entropy interface.
 
-Health tests, when enabled in the {{< regref "ES_CONF" >}} register, will run continuously.
-Two tests are implemented in hardware, the repetition count test (RCT), and the adaptive proportion test (APT).
-
-There are four health test blocks, one per inbound rng bus bit.
-Each health test block works independently from one another.
-The checks are done on the serial stream input, and an optional feature allows for passing entropy from a single bit stream source.
-Once the streams have passed the health tests, it becomes available for downstream consumption.
-If the health tests ever fail, the ES FIFO is prevented from receiving additional entropy bits up.
-The entropy health tests will continue until a clean stream is detected, at which time the ES FIFO will start collecting bits again.
-The {{< regref "ES_RCT_HEALTH" >}} register and the  {{< regref "ES_APT_HEALTH" >}} register set the cutoff limits for the health tests.
-
-The above process will be repeated for as long as entropy bits are to be collected.
-At any time, the enable bit can be cleared, and the entropy generation will halt immediately.
+After a reset, the ENTROPY_SRC block will start up in bypass mode by default.
+This feature is designed to provide an initial seed's worth of entropy at a faster rate than the normal FIPS/CC compliant entropy process.
+Health testing will still be done on bypass mode entropy, but the window of checking is, by default, 384 bits instead of 1024 bits.
+When entropy is delivered to the downstream hardware block, a signal will indicate what type of entropy it is - FIPS compliant or not.
+Bypass mode can be completely disabled in the {{< regref "CONF" >}} register.
+
+Once the initial bypass mode phase has completed, the ENTROPY_SRC block will switch to FIPS compliant mode. 
+In this mode, once the raw entropy has been health checked, it will be passed into a conditioner block.
+This block will compress the bits such that the entropy bits/physical bits, or min-entropy value, should be improved over the raw data source min-entropy value.
+The compression operation, by default, will be 832 bits out of 1024 tested bits to be compressed into 384 bits.
+
+Health checks are done on the input raw data from the digital noise source.
+There are four health tests that will be performed: repetitive count, adaptive proportion, bucket, and markov tests.
```

Comment:
markov -> Markov

Reply:
Capitalization changed.  As far as wordsmithing goes, I have a lot of ideas.  Though I think it would be easier to do that in my own PR.  I'm going through this for design accuracy right now, but I'll go through it for statistical completeness in an future PR.